### PR TITLE
support Python 3.5

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

File: docker/utils/utils.py [hunk 1]
```diff
@@ -546,12 +546,6 @@ def datetime_to_timestamp(dt):
     return delta.seconds + delta.days * 24 * 3600
 
 
-def longint(n):
```
Comment (graingert) [inline docker/utils/utils.py]: all this does is allow users to enter their storage size in pounds (300lb) on python 2:

File: tests/unit/utils_test.py [hunk 2]
```diff
@@ -299,56 +299,30 @@ def test_convert_volume_binds_no_mode(self):
         self.assertEqual(convert_volume_binds(data), ['/mnt/vol1:/data:rw'])
 
     def test_convert_volume_binds_unicode_bytes_input(self):
-        if six.PY2:
```
Comment (dnephin) [inline tests/unit/utils_test.py]: I don't think we want to remove the tests for PY2. Having them in separate test cases with `pytest.mark.skip` would be fine.

Comment (graingert) [inline tests/unit/utils_test.py]: They're still there. I just combined the 3K and 2 tests

Comment (dnephin) [inline tests/unit/utils_test.py]: Oh, I missed that, sounds good.

**Comment by graingert** [comment]
@dnephin any update on a merge?

**Comment by shin-** [comment]
I thought I had commented on there, sorry. Changes LGTM.

**Comment by dnephin** [comment]
LGTM

**Pull request merged**